### ISSUE TITLE
Wrong metrics port in eventing services installed with 1.9.0 and eventing v0.15.2 

### ISSUE THREAD

**Issue by skonto** [openshift-knative/serverless-operator#435]
Running `make images install` on master, on crc 4.5.1 I get (installs 1.9.0 by default):
```
$ oc describe svc broker-ingress -n knative-eventing
Name:              broker-ingress
Namespace:         knative-eventing
Labels:            eventing.knative.dev/brokerRole=ingress
                   eventing.knative.dev/release=v0.15.2
Annotations:       manifestival: new
Selector:          eventing.knative.dev/brokerRole=ingress
Type:              ClusterIP
IP:                172.25.230.58
Port:              http  80/TCP
TargetPort:        8080/TCP
Endpoints:         10.116.0.156:8080
Port:              http-metrics  9090/TCP
TargetPort:        9090/TCP
Endpoints:         10.116.0.156:9090
Session Affinity:  None
Events:            <none>
[stavros@easy apisource]$ oc describe svc broker-filter -n knative-eventing
Name:              broker-filter
Namespace:         knative-eventing
Labels:            eventing.knative.dev/brokerRole=filter
                   eventing.knative.dev/release=v0.15.2
Annotations:       manifestival: new
Selector:          eventing.knative.dev/brokerRole=filter
Type:              ClusterIP
IP:                172.25.34.67
Port:              http  80/TCP
TargetPort:        8080/TCP
Endpoints:         10.116.0.155:8080
Port:              http-metrics  9090/TCP
TargetPort:        9090/TCP
Endpoints:         10.116.0.155:9090
Session Affinity:  None
Events:            <none>
```
However when I do:
```
$ oc get pods -n knative-eventing
NAME                                       READY   STATUS      RESTARTS   AGE
broker-controller-588cc5c794-mhfzf         1/1     Running     0          17m
broker-filter-6d55dbc758-znvv6             1/1     Running     0          17m
broker-ingress-577d68fb9f-2cjgq            1/1     Running     0          17m
eventing-controller-557467d646-h54ld       1/1     Running     0          17m
eventing-webhook-7bcb5c78d7-5zt7j          1/1     Running     0          17m
imc-controller-cb77b849c-kgfcd             1/1     Running     0          17m
imc-dispatcher-685b75dbf9-fxmjf            1/1     Running     0          17m
mt-broker-controller-6c8f6dd8f9-jldwd      1/1     Running     0          17m
pingsource-mt-adapter-6bd5bb645f-cphbt     1/1     Running     0          17m
storage-version-migration-eventing-cssh4   0/1     Completed   0          17m
v0.15.0-upgrade-db8tn                      0/1     Completed   0          17m

$ oc port-forward broker-filter-6d55dbc758-znvv6  -n knative-eventing 9092
Forwarding from 127.0.0.1:9092 -> 9092
Forwarding from [::1]:9092 -> 9092
Handling connection for 9092
Handling connection for 9092

$ curl localhost:9092/metrics
# HELP mt_broker_filter_client_latency How long Kubernetes API requests take
# TYPE mt_broker_filter_client_latency histogram
mt_broker_filter_client_latency_bucket{name="",le="1e-05"} 0
mt_broker_filter_client_latency_bucket{name="",le="0.0001"} 0
mt_broker_filter_client_latency_bucket{name="",le="0.001"} 0
mt_broker_filter_client_latency_bucket{name="",le="0.01"} 1
mt_broker_filter_client_latency_bucket{name="",le="0.1"} 1
mt_broker_filter_client_latency_bucket{name="",le="1"} 2
mt_broker_filter_client_latency_bucket{name="",le="10"} 2
mt_broker_filter_client_latency_bucket{name="",le="100"} 2
mt_broker_filter_client_latency_bucket{name="",le="+Inf"} 2
....
$ curl localhost:9090/metrics
curl: (7) Failed to connect to localhost port 9090: Connection refused

Same for broker-ingress-577d68fb9f-2cjgq.
```

I see that under config dir in the release branch of 0.15.2 in knative eventing 9090 is used:
https://github.com/knative/eventing/blob/v0.15.2/config/brokers/mt-channel-broker/deployments/broker-filter.yaml,
however the default metrics port is 9092 on the same tag: https://github.com/knative/eventing/blob/115a31ad2b92c928a2f4484884a38ff71bb53161/cmd/broker/filter/main.go#L56.
In later releases this is updated: https://github.com/knative/eventing/blob/v0.16.0/config/brokers/mt-channel-broker/deployments/broker-filter.yaml.
This is probably an eventing bug but wanted to be sure it is not a downstream release issue so I opened it here first.
Also it affects our release anyway.

This blocks the work for adding the eventing dashboard because the service monitor depends on it.

**Comment by matzew** [comment]
@skonto it was change to `9092` for 0.16+

if you want to change, I guess the best is to just update this file:
https://github.com/openshift-knative/operator/blob/release-0.15/cmd/operator/kodata/knative-eventing/0.15.2/1-eventing.yaml

(our fork of the operator, shipping 0.15)

**Comment by skonto** [comment]
thanks @matzew I will create a PR.

**Comment by markusthoemmes** [comment]
@skonto is this fixed via your PR?

**Comment by skonto** [comment]
yes we can close it.

**Comment by markusthoemmes** [comment]
@skonto any news?

**Comment by skonto** [comment]
Checking now.

**Comment by skonto** [comment]
The mismatch is fixed after v0.18: https://github.com/knative/eventing/commit/975a516517c0198d354e72388da2bdf024dbef36#diff-f9e65ebfc374aa8dd9a738680c99bbe91313020aa4257a71a95034f309f131fe

**Comment by skonto** [comment]
@markusthoemmes feel free to close this one.
